specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -216,6 +269,24 @@
           }
         }
       }
+    },
+    "RoutingForNumber": {
+      "description": "Represents number routing validation details.",
+      "type": "object",
+      "properties": {
+        "targetNumber": {
```

Comment:
Why is this included in the response? the `targetNumber` is required parameter in the API request, so client already has the target number being queried.

Request parameters should only be included in responses in exceptional cases.